### DISCUSSION TITLE
Sveltekit 415 not works

### DISCUSSION THREAD

**Discussion by bitdom8** [ivanhofer/typesafe-i18n#426] [General]
Translations (using this repo https://github.com/ivanhofer/typesafe-i18n/tree/main/packages/adapter-svelte) are not appearing after migrating new version: https://github.com/sveltejs/kit/discussions/5774

**Comment by ivanhofer** [comment]
please take a look at the demo application: https://github.com/ivanhofer/typesafe-i18n-demo-sveltekit

**Reply by bitdom8** [reply]
Thanks @ivanhofer  . All good now
